.github/workflows/deploy-my-kibana.yml

```diff
@@ -17,14 +17,35 @@ on:
 
 permissions:
   contents: read
+  # if you listen for PRs, use this to use some comment reactions
+  pull-requests: write
 
 jobs:
+  trim-github-comment:
```

Comment:
it provision always a runner with any comment event

Reply:
No. only if it contains `/oblt-deploy` , therefore this can help with the edge cases

Reply:
But you are requesting to process any comment completely so, for example the following comment would be processed to do nothing

```
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
```

Reply:
Also, IIRC, each runner has a minimum usage time of 1 minute; due to you using two, we will duplicate the time usage.

Reply:
> But you are requesting to process any comment completely so, for example the following comment would be processed to do nothing


No. That's not possible. `/oblt-deploy` should be in the github comment, see `L25`


```
if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, '/oblt-deploy') }} 
```

If we consider trimming as a feature to help with the DX, then we need to pay for the usage with this hack. However, IIUC, GitHub runners in public repositories are not an issue.

I don't think the hack will be much of a concern, except for https://github.com/elastic/kibana/blob/main/.github/workflows/oblt-github-commands.yml. Which will require to exclude any comments produced by any bot

Reply:
I need to exclude comments like:
- https://github.com/elastic/kibana/pull/203712#issuecomment-2541466050